angular-ngrx-scss/src/app/home/profile/profile-about/profile-about.component.html

```diff
@@ -5,7 +5,7 @@
     [src]="profile?.user?.avatar"
   />
 
-  <div class="content">
+  <div class="content" *ngIf="profile?.user">
```

Comment:
suggestion: Since we're checking if `profile.user` is defined here, we no longer need to check inside the div for each field individually anymore. For example the name usage `profile?.user?.name` can be changed to `profile.user.name` now, and any similar fields can also be changed in a similar manner.